test-qemu/port_forwarding_test.go

```diff
@@ -59,6 +59,36 @@ var _ = ginkgo.Describe("port forwarding", func() {
 		gomega.Expect(string(out)).To(gomega.ContainSubstring("Hello from the host"))
 	})
 
+	ginkgo.It("should reach a http server on the host", func() {
```

Comment:
The first commit apparently does a bit more than what is in the commit log, this test seems to be new, and the commit also adds support for uncompressing gz files (in addition to xz files)

Reply:
updated. I moved the decompression func to the second commit when it is actually used. Regarding this test, it was a duplicate so I just removed it. I think I copy/paste the one above to create a new one starting from that and then I forgot.